src/gaurav/Banking.java

```diff
@@ -0,0 +1,149 @@
+package gaurav;
+
+public class Banking {
+	static int iciciBalance =10000;
+	static int hdfcBalance =15000;
+	static int bobBalance =14000;
+	static int sbiBalance =16000;
+	
+	static int totalDebitCount;
+	static int totalCreditCount;
+	
+	int iciciCreditCount=0, hdfcCreditCount=0,bobCreditCount=0 ,sbiCreditCount=0;
+	int iciciDebitCount=0, hdfcDebitCount=0,bobDebitCount=0 ,sbiDebitCount=0;
+    
+	
+	void operation(String bank,String op,int amt){
+		switch(op){
+			case "debit":
+				switch(bank){
+					case"icici":
+						if(iciciBalance>=amt) {
```

Comment:
instead of writing same code block for multiple times, you can write debit() method which accept the two arguments and you can simply call that method.